### PR TITLE
Release/7.1.0

### PR DESCRIPTION
**Pull request by tonidero**
### Other Changes
* Fix missing gitignore causing release process errors via Toni Rico (@tonidero)
### Dependency Updates
* [AUTOMATIC BUMP] Updates purchases-hybrid-common to 7.3.0 (#142) via RevenueCat Git Bot (@RCGitBot)
    * [Android 7.2.0](https://github.com/RevenueCat/purchases-android/releases/tag/7.2.0)
    * [iOS 4.30.2](https://github.com/RevenueCat/purchases-ios/releases/tag/4.30.2)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -67,3 +67,6 @@ yarn.lock
 # fastlane
 report.xml
 fastlane/.env
+
+# CI dependencies
+vendor
```
Comment (tonidero) [inline .gitignore:72]: This fixes the issue in #143

**Comment by tonidero** [comment]
Did this manually, and should be ready now.

**Review by NachoSoto — APPROVED** [review]

**Comment by NachoSoto** [comment]
Yay awesome!

**Comment by NachoSoto** [comment]
Good to deploy?

**Comment by tonidero** [comment]
Yup! Started deploy

**Comment by NachoSoto** [comment]
Awesome! Capacitor was a bit behind, so glad we're all up to date now 🙏🏻

**Pull request merged**